### PR TITLE
Feature/add unstable participant api

### PR DESCRIPTION
**Pull request by youliangtan**
- Related to `adpater.RobotUpdateHandle.get_unstable_participant()`. 
- Also traj creation `adpater.schedule.make_trajectory()`. 

Note: pending fix on bad shutdown after `Participant.set_itinerary()`

### CODE REVIEW

**Comment by Yadunund** [comment]
Could we add bindings for `set_charger_waypoint()`?
https://github.com/osrf/rmf_core/blob/5b6631686fe48554fb9b68900a66f5696805d85e/rmf_fleet_adapter/include/rmf_fleet_adapter/agv/RobotUpdateHandle.hpp#L91-L94

**Comment by youliangtan** [comment]
> Could we add bindings for `set_charger_waypoint()`?
> https://github.com/osrf/rmf_core/blob/5b6631686fe48554fb9b68900a66f5696805d85e/rmf_fleet_adapter/include/rmf_fleet_adapter/agv/RobotUpdateHandle.hpp#L91-L94

Added: https://github.com/osrf/rmf_fleet_adapter_python/pull/11/commits/e8f4fd59bccfc8041f21c3af6ec365f86cdde866

File: rmf_fleet_adapter_python/src/graph/graph.cpp [hunk 1]
```diff
@@ -168,6 +168,10 @@ void bind_graph(py::module &m) {
       .def("get_lane", py::overload_cast<std::size_t>(&Graph::get_lane))
       .def("get_lane", py::overload_cast<std::size_t>(
           &Graph::get_lane, py::const_))
+      .def("lane_from", py::overload_cast<std::size_t, std::size_t>(
```
Comment (Yadunund) [inline rmf_fleet_adapter_python/src/graph/graph.cpp:171]: Can we also add [this](https://github.com/osrf/rmf_core/blob/163d700b2a61fb4862f69840601e7e1471e39091/rmf_traffic/include/rmf_traffic/agv/Graph.hpp#L506-L507) overloaded method?
Lets call it `lanes_from_waypoint(std::size_t)`

Comment (youliangtan) [reply rmf_fleet_adapter_python/src/graph/graph.cpp:171]: https://github.com/osrf/rmf_fleet_adapter_python/pull/11/commits/deb4ac601e2b8b3ec7175630c049a12180d38503

**Review by Yadunund — APPROVED** [review]

**Pull request merged**